hare/flows_test.go

```diff
@@ -405,35 +251,42 @@ func Test_multipleCPs(t *testing.T) {
 
 	test.initialSets = make([]*Set, totalNodes)
 
-	dbs := make([]*sql.Database, 0, totalNodes)
-	for i := 0; i < totalNodes; i++ {
-		dbs = append(dbs, sql.InMemory())
-	}
-	pList := make(map[types.LayerID][]types.ProposalID)
+	pList := make(map[types.LayerID][]*types.Proposal)
 	for j := types.GetEffectiveGenesis().Add(1); !j.After(finalLyr); j = j.Add(1) {
-		for i := uint64(0); i < 200; i++ {
```

Comment:
If I run either of the `Test_multiple...` tests with your changes but the old value of 200 here they timeout

Reply:
yes. like i explained in the PR description, assembling 200 proposals from the mock and get them ready caused ~2sec delay and cause some of the nodes to end preround with empty set, because by the time they get to preround, preround is already over.

Reply:
:+1: sorry!